src/api/config/environments/development.rb

```diff
@@ -8,8 +8,8 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  # Do not eager load code on boot.
-  config.eager_load = false
+  # Eager load code on boot.
+  config.eager_load = true
```

Comment:
A note on this change: because I added a folder `app/controllers/webui/project/` it meant that any controller code within the `app/controllers/webui` would look for a module in that folder whenever they called the `Project` model. I think its because of rails autoloading not correctly resolve the constant. However enabling eager_load in development fixes this problem. The only downside is that its said to increase startup times for the rails server but I haven't noticed a difference myself.

There are some other solutions in case we dont want to enable eager_load:

1. Add `require 'project'` in the `WebuiController` - I'm not such a fan of this because it doesn't fix the problem for other developers who may add another folder to the webui controllers in the future i.e. adding `app/controllers/webui/package/` would still cause an "Unitialised constant" error and cause that developer to spend time trying to figure out the solution unless they happen to already know about this.

2. Change all calls to the `Project` model in the webui controllers to `::Project` - I think this is a viable solution but also more work..